src/ThreeJsPanel.js

```diff
@@ -67,6 +67,7 @@ export class ThreeJsPanel {
         // set pixel ratio to 0.25 or 0.5 to render at lower res.
         this.renderer.setPixelRatio(window.devicePixelRatio);
         this.renderer.state.setBlending(NormalBlending);
+        this.renderer.localClippingEnabled = true;
```

Comment:
Probably nit picking but is this something that can't be set under line 64 when you're constructing `this.renderer`?

Reply:
Glad you made me look at this -- I think this same line needs to be added in the block below here also.  I'll see if it's allowed in the properties passed in to the constructor too.

Reply:
The threejs docs don't list this as an allowed parameter for the constructor of the WebGLRenderer.  But I did move it down so that it is applied to both WebGL 1 and WebGL2 cases.